test/static/es6-support.js

```diff
@@ -46,3 +47,82 @@ class A {
     this.data = 'Hello World';
   }
 }
+
+/**
+ * I am a description of test-element.
+ *
+ * @hero /path/to/hero.png
```

Comment:
No, it's cargo-cult copied from another test - I like having it in to make sure it doesn't throw or cause strange behavior, but I can reduce this case significantly.